### PR TITLE
Bump golangci-lint version in Github Workflow

### PR DESCRIPTION
**Pull request by linouxis9**
Fix issue mentioned in https://github.com/wmnsk/go-tcap/pull/17#issuecomment-2022481870

### CODE REVIEW

File: .github/workflows/golangci-lint.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-          version: v1.33
+          version: v1.57.1
```
Comment (wmnsk) [inline .github/workflows/golangci-lint.yml]: I'd like to use the `latest`  instead.

```suggestion
          version: "latest"
```

**Review by wmnsk — COMMENTED** [review]
good catch, and thanks! would it be problematic to use `latest` which I prefer?

**Comment by linouxis9** [comment]
Hi,
Yes I completely agree, it's better to stick with latest than stumbling upon the same issue in a few months / years.
Thanks!

**Review by wmnsk — APPROVED** [review]
👍

**Pull request merged**